arangosh/Utils/ManagedDirectory.cpp

```diff
@@ -245,7 +245,7 @@ ManagedDirectory::ManagedDirectory(application_features::ApplicationServer& serv
       return;
     }
 
-    std::vector<std::string> files(TRI_FullTreeDirectory(_path.c_str()));
+    std::vector<std::string> files(TRI_FilesDirectory(_path.c_str()));
     bool isEmpty = (files.size() <= 1);
     // TODO: TRI_FullTreeDirectory always returns at least one element ("")
     // even if directory is empty?
```

Comment:
And this comment is now obsolete, because we are not using `TRI_FullTreeDirectory` anymore.